ui_src/src/components/createStationForm/index.js

```diff
@@ -62,70 +65,74 @@ const storageOptions = [
     }
 ];
 
-const CreateStationForm = ({ createStationFormRef }) => {
-    const [getStartedState, getStartedDispatch] = useContext(GetStartedStoreContext);
+const CreateStationForm = ({ createStationFormRef, getStartedStateRef, finishUpdate, updateFormState, getStarted, setLoading }) => {
+    const history = useHistory();
     const [creationForm] = Form.useForm();
     const [allowEdit, setAllowEdit] = useState(true);
+    const [actualPods, setActualPods] = useState(null);
+    const [retentionType, setRetentionType] = useState(retanionOptions[0].value);
+    const [storageType, setStorageType] = useState(storageOptions[0].value);
 
     useEffect(() => {
-        checkState();
-        createStationFormRef.current = onFinish;
+        getOverviewData();
```

Comment:
Call it just if you are in get-started mode.

Reply:
in getOverviewData I receive the number of pods in order to get the maximal number of replicas. I need it in both modes.